jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -367,7 +379,14 @@ def _jwst2data(file_obj, ext, data_label):
 
             # This is instance of gwcs.WCS, not astropy.wcs.WCS
             if 'wcs' in dm_meta:
-                data.coords = dm_meta['wcs']
+                gwcs = dm_meta['wcs']
+
+                if try_gwcs_to_fits_sip:
+                    coords = _try_gwcs_to_fits_sip(gwcs)
```

Comment:
If you want FITS WCS anyway, can we just grab it from the FITS extension? The only reason I was accessing the ASDF-in-FITS was for the GWCS. By contract, JWST is required to also ship the FITS WCS in normal FITS header (even if it cannot represent all the distortion, or something like that), so maybe this conversion is not necessary.

Reply:
Good point. I'll simplify this part of the parser to use FITS as FITS.